4-DeviceCodeFlow/Console-DeviceCodeFlow-v2/Program.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using System;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Console_DeviceCodeFlow_MultiTarget
+{
+    internal class Program
+    {
+        private static PublicClientApplicationOptions appConfiguration = null;
+        private static IConfiguration configuration;
+        private static string MSGraphURL;
+       
+        // The MSAL Public client app
+        private static IPublicClientApplication application;
+
+        private static async Task Main(string[] args)
+        {
+            // Using appsettings.json to load the configuration settings
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(System.IO.Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json");
+
+            configuration = builder.Build();
+
+            appConfiguration = configuration.Get<PublicClientApplicationOptions>();
+
+            MSGraphURL = configuration.GetValue<string>("GraphApiUrl");
+
+            // Sign-in user using MSAL and obtain an access token for MS Graph
+            GraphServiceClient graphClient = await SignInAndInitializeGraphServiceClient(appConfiguration);
+
+            // Call the /me endpoint of MS Graph
+            await CallMSGraph(graphClient);
+        }
+
+        private static async Task<string> SignInUserAndGetTokenUsingMSAL(PublicClientApplicationOptions configuration)
```

Comment:
would we want to pass the scopes as parameters instead of hardcoding them in the method?
See also my comments in https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/pull/9

Reply:
I've declared the scope in Main and passed it as parameter in following methods. Please review.
Thanks!